content/post/kubedb-v2024.02.14/index.md

```diff
@@ -0,0 +1,607 @@
+            name: rd-auth1
+            items:
+              - key: username
+                path: auth/1/username
+              - key: password
+                path: auth/1/password
+        - secret:
+            name: rd-auth2
+            items:
+              - key: username
+                path: auth/2/username
+              - key: password
+                path: auth/2/password
+        - configMap:
+            name: redis-init-script
+  terminationPolicy: WipeOut
+```
+After successful deployment two given users using secrets `rd-auth1` and `rd-auth2` are created along with `default` user.
+
+## Postgres
+
+### Grafana Alerts Dashboard
+In this release, we bring support for Postgres grafana summary dashboard with alerts for a specific Postgres instance.
+
+To use the alert dashboards :
+
+* First you need to have KubeDB operator and a Postgres instance running on your cluster along with Prometheus operator or Builtin Prometheus supported by Kubedb.
+
+* Then create a file named overwrite.yaml having the following values. Make sure you change the values matching your needs.
+
+    - For prometheus operator
+        ```yaml
+        resources:
+          - postgres
+
+        dashboard:
+          folderID: 0
+          overwrite: true
+          alerts: true 
+          replacements: []
+
+        grafana:
+          version: “provide your grafana version”
+          url: "provide your grafana service dns" # example: http://grafana.monitoring.svc:80 
+          apikey: "provide grafana apikey"
+
+        app:
+          name: “provide your postgres instance name”
+          namespace: “provide the namespace”
+        ```
+    - For prometheus builtin
+       ```yaml
+        resources:
+          - postgres
+
+        dashboard:
+          folderID: 0
+          overwrite: true
+          alerts: true 
+        replacements:
+          job=\"kube-state-metrics\": job=\"kubernetes-service-endpoints\"
+          job=\"kubelet\": job=\"kubernetes-nodes-cadvisor\"
+          job=\"$app-stats\": job=\"kubedb-databases\"
+
+        grafana:
+          version: “provide your grafana version”
+          url: "provide your grafana service dns" # example: http://grafana.monitoring.svc:80 
+          apikey: "provide grafana apikey"
+
+        app:
+          name: “provide your postgres instance name”
+          namespace: “provide the namespace”
+        ```    
+
+* now follow the steps to install grafana dashboards:
+    ```bash
+    $ helm repo add appscode https://charts.appscode.com/stable/
+    $ helm repo update
+    $ helm search repo appscode/kubedb-grafana-dashboards --version=v2024.1.31
+    $ helm upgrade -i kubedb-grafana-dashboards appscode/kubedb-grafana-dashboards -n kubeops --create-namespace --version=v2024.1.31 
+    -f  /path/to/the/overwrite.yaml
+    ```
+
+### Bug fix and improvements
+* We have fixed a bug in VersionUpdate OpsRequest for Postgres, which was causing issues on updating major versions .
```

Comment:
Improved the archiver feature. Chnaged the wal directory

Reply:
Breaking changes: 

We have made the volumeExpansion mode required. `ops.volumeExpansion.mode`. Not defaulting it now.

Reply:
Updated the mongodb exporter images